tools/serverpod_cli/lib/src/generator/open_api/open_api_extensions.dart

```diff
@@ -0,0 +1,68 @@
+// ignore_for_file: unused_element
+
+part of openapi_definition;
+
+extension on String {
```

Comment:
There may be support for this in some imported package, it would be great to double check so that we don't duplicate any code.

Reply:
@SandPod  I can't find that implementation on super_string